test/ssc_test/cmod_windpower_test.cpp

```diff
@@ -30,7 +30,8 @@ TEST_F(CMWindPowerIntegration, HubHeightInterpolation_cmod_windpower) {
     ssc_data_get_number(data, "annual_energy", &annual_energy);
     EXPECT_GT(annual_energy, 4e06) << "Annual energy should be higher than height at 90";
 
-    free_winddata_array(windresourcedata);
+//    delete[] windresourcedata;
```

Comment:
Why is this free no longer required?

Reply:
The free was moved to the test tear down.

Reply:
reverted to develop tests